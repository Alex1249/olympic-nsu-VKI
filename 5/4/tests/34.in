26
h i v k z y w x l e b n t m u r o d f c j q s a p g
