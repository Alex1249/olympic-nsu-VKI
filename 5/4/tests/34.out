h i v k z y w x l e b n t m u r o d f c j q s g a p
